src/schemathesis/cli/__init__.py

```diff
@@ -493,6 +509,8 @@ def run(
     app: Optional[str] = None,
     request_timeout: Optional[int] = None,
     request_tls_verify: bool = True,
+    request_cert: Optional[str] = None,
```

Comment:
Is it the right type? I guess it should be `Optional[click.utils.LazyFile]` - at least I am not aware that `click` automatically reads files in such cases.

Reply:
Fixed.